nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -0,0 +1,43 @@
+collision_monitor_node:
+  ros__parameters:
+    use_sim_time: True
+    base_frame_id: "base_footprint"
+    odom_topic: "/odom"
+    polygon_topic: "polygons"
+    cmd_vel_in_topic: "/cmd_vel_raw"
+    cmd_vel_out_topic: "/cmd_vel"
+    transform_tolerance: 0.1
+    max_time_shift: 10.0
+    simulation_time_step: 0.02
+    release_step: 0.002
```

Comment:
This is increase in velocity when releasing the robot from APPROACH model (also refernced in https://github.com/ros-planning/navigation2/pull/2982#discussion_r888935114). This is a increase step made on each `processMain()` cycle. If we are targeting 50Hz frequency for Turtlebot3, it will be `+0.1 meter/s` velocity increase per each second, which seems to be rational for TB3, which maximum velocity is `0.26`. This means, robot will return to its normal operation after ~2.5 seconds after full stop during approach model (if any).

This value was also adjusted in run-time simulation experiments, higher values will cause robot to twitch more notable.

Therefore I believe, for TB3 this is reasonable (at least in order of magnitude) value.

Reply:
I think this needs to be flushed out more, I don't think we can release with this kind of feature in place unless its massaged into something more human understandable or fixing the underlying problem this is addressing

Reply:
Agree with you. Replaced this parameter to `release_acceleration` which is `0.1 m/s^2` by default. I think, it makes more physical sense.

Reply:
That makes much more sense! I remember at some other time in development you mentioned you added this to resolve some other issue. Assuming that there's some velocity smoother after this that will deal with making sure that the commands sent to the base controller are within kinematic limits, why have this here? Is this still fixing some other problem? Else, we can safely remove since the `velocity smoother` will be added to the default configuration right below this node's output.